src/BaseSphere.cpp

```diff
@@ -80,8 +80,12 @@ void BaseSphere::DrawAtmosphereSurface(Graphics::Renderer *renderer,
 
 	renderer->SetTransform(matrix4x4f(modelView * matrix4x4d::ScaleMatrix(rad) * invrot));
 
-	if (!m_atmos)
-		m_atmos.reset(new Drawables::Sphere3D(renderer, 4, 1.0f, ATTRIB_POSITION));
+	if (!m_atmos) {
+		// 5 subdivision = 20472 verts, 61440 indices NB: this should be configurable!
+		static constexpr int subdivisions = 5;
+		m_atmos.reset(new Drawables::Sphere3D(renderer, subdivisions, 1.0f, ATTRIB_POSITION));
```

Comment:
Note: I would strongly prefer the number of subdivisions used for planets be controlled in part by the planet detail setting. We probably don't want to be wasting bandwidth on ~5,000 triangles of atmosphere mesh (15k indices) when significantly fewer would suffice on Raspberry Pi, for example.

Reply:
phew, ok that was an adventure but I've done it, High and Very High get subdivs 5, everything else gets 4